Makefile

```diff
@@ -113,3 +114,24 @@ submodules:
 	@git submodule update --init --recursive
 
 .PHONY: manifests cobertura reviewable submodules fallthrough
+
+# ====================================================================================
+# Special Targets
+
+define CROSSPLANE_HELP
+Crossplane Targets:
+    manifests          Generate manifests e.g. CRD, RBAC etc.
+    cobertura          Generate a coverage report for cobertura applying exclusions on generated files.
+    reviewable         Ensure a PR is ready for review.
+    test-integration   Run integration tests.
```

Comment:
To be consistent with the build repository targets, please use `e2e`.
Please see the change here
https://github.com/crossplaneio/crossplane/pull/635

Reply:
Nice! Thanks

Reply:
I'd remove the `test-integration` target from the help, if `e2e` is already being printed out.

Reply:
Will do! Thanks for your feedback and implementation of the integration tests :)